server/application-server/src/main/java/de/tum/in/www1/hephaestus/codereview/pullrequest/PullrequestController.java

```diff
@@ -0,0 +1,28 @@
+package de.tum.in.www1.hephaestus.codereview.pullrequest;
+
+import java.util.List;
+
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RestController;
+
+@RestController
+@RequestMapping("/pr")
```

Comment:
Github names this mapping `pulls` maybe name it `pullrequests` or `pullrequest`? But if you want to keep `pr` that's also fine